build/tools/travis-ci.sh

```diff
@@ -54,7 +54,14 @@ case $wxTOOLSET in
         ;;
     *)
         echo 'Configuring...' && echo -en 'travis_fold:start:script.configure\\r'
-        ./configure --disable-optimise $wxCONFIGURE_FLAGS
+        ./configure --disable-optimise --disable-debug_info $wxCONFIGURE_FLAGS || rc=$?
+        if [ "$rc" != 0 ]; then
```

Comment:
Still not right. How about this:
```
./configure --disable-optimise --disable-debug_info $wxCONFIGURE_FLAGS
rc=$?
if [ $rc != 0 ]; then
```
```

Reply:
Ahh, no, there is a `set -e`. Maybe this:
```
if [ -n "$rc" ]; then
```

Reply:
Yes, this is what I ended up by doing, sorry for the flurry of commits.

I still have no idea why does GTK 3 STL build fail though.